metricproviders/datadog/datadogV1_test.go

```diff
@@ -33,8 +33,9 @@ func TestRunSuite(t *testing.T) {
 
 	ddProviderInterval10m := v1alpha1.MetricProvider{
 		Datadog: &v1alpha1.DatadogMetric{
-			Query:    "avg:kubernetes.cpu.user.total{*}",
-			Interval: "10m",
+			Query:      "avg:kubernetes.cpu.user.total{*}",
+			Interval:   "10m",
+			ApiVersion: "v1",
```

Comment:
Can we have a test that tests the default value as well, so that when it changes tests will also need updating etc.

Reply:
I am ok even just dropping setting v1 hear and only testing via the default path.

Reply:
Commit: https://github.com/argoproj/argo-rollouts/pull/2592/commits/79112123f09c1e60d96227e595cb85aa0679e0e5